### PR TITLE
Add entrypoint script

### LINKED ISSUE #43: Add ENTRYPOINT to the web app image
In order to make it easier to deploy and run the image, it should include the default entrypoint to the application

### PR DESCRIPTION
**Pull request by fmrsabino**
Closes #43 

- Adds a Docker `ENTRYPOINT` to the production image (also applies to the dev image).
- The entrypoint performs the following:
  * Executes migrations
  ~* Loads initial fixture data using `manage.py loaddata` (this does not override current data)~
  * Launches app with `Gunicorn`

### CODE REVIEW

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/ash
+
+set -euo pipefail
+
+echo "==> $(date +%H:%M:%S) ==> Migrating Django models..."
+python src/manage.py migrate --noinput
+
+echo "==> $(date +%H:%M:%S) ==> Applying fixtures..."
+python src/manage.py loaddata initial_default_apps.json
```
Comment (Uxio0) [inline docker-entrypoint.sh:9]: Will this be removed in next versions?

**Review by Uxio0 — APPROVED** [review]

File: docker-entrypoint.sh (hunk 1 above)
Comment (fmrsabino) [reply docker-entrypoint.sh:9]: Do you think it's better for us to load the fixture independently of the deployment? I can remove it on the context of this PR

Comment (fmrsabino) [reply docker-entrypoint.sh]: Removed here 46cc4dc

Comment (Uxio0) [reply docker-entrypoint.sh]: Yes, I think so. If not we can find surprises if we forget about it

**Review by Uxio0 — APPROVED** [review]

**Pull request merged**